src/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0 auto;
+  padding: 0 auto;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  position: relative;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  position: relative;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars_star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--1 .stars_star:nth-child(1)::after,
+.stars--2 .stars_star:not(:nth-child(n+3))::after,
+.stars--3 .stars_star:not(:nth-child(n+4))::after,
+.stars--4 .stars_star:not(:nth-child(n+5))::after,
+.stars--5 .stars_star:not(:nth-child(n+6))::after {
+  content: "";
+  position: absolute;
+  background-image: url(images\star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
As Joy said, you don't need to involve a pseudo element here. Simply change the background image of a star